README.md

```diff
@@ -5031,7 +5031,9 @@ apache::vhost { 'sample.example.net':
   docroot     => '/path/to/directory',
   directories => [
     { path  => '/path/to/different/dir',
-      additional_includes => [ '/custom/path/includes', '/custom/path/another_includes', ],
+      additional_includes => ['/custom/path/includes',
```

Comment:
I'd actually consider keeping this on the same line.

Reply:
Just did that, thanks!